package.json

```diff
@@ -1,58 +1,67 @@
 {
   "name": "vue-dictaphone",
   "description": "Vue.js dictaphone component to record audio from the user",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "author": {
     "name": "Manuel Wieser",
     "email": "office@manuelwieser.com",
     "url": "https://manu.ninja/"
   },
+  "main": "dist/vue-dictaphone.umd.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/Lorti/vue-dictaphone.git"
   },
   "license": "MIT",
   "scripts": {
-    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.dev.conf.js",
-    "test": "jest --config test/unit/jest.conf.js",
-    "build": "cross-env NODE_ENV=production webpack --config build/webpack.prod.conf.js"
+    "serve": "vue-cli-service serve src/example/main.js",
+    "build": "vue-cli-service build --target lib src/index.js",
```

Comment:
🙇 
Could you please add "prepublish": "npm run build" also?

Reply:
:heavy_check_mark: